spec/functional/command/repo/update_spec.rb

```diff
@@ -39,6 +39,16 @@ module Pod
       (repo2 + 'README').read.should.include 'Updated'
     end
 
+    # Conditionally skip the test if `tmutil` is not available.
+    has_tmutil = system('tmutil version', :out => File::NULL)
```

Comment:
shoudl not be using a shell here

Reply:
Thanks, changed to `system('tmutil', 'version')`.